oneflow/core/job/runtime.cpp

```diff
@@ -70,9 +70,10 @@ Runtime::Runtime(
     Singleton<RuntimeJobDescs>::Get()->AddPlan(plan);
     collective_boxing_scheduler_plan_token_ =
         Singleton<boxing::collective::Scheduler>::Get()->AddPlan(plan);
-#ifdef WITH_CUDA
+    // #ifdef WITH_CUDA
+    //     Singleton<EagerCclCommMgr>::Get()->CreateCommFromPlan(plan);
+    // #endif  // WITH_CUDA
     Singleton<EagerCclCommMgr>::Get()->CreateCommFromPlan(plan);
```

Comment:
`Thread 1 "pt_main_thread" received signal SIGSEGV, Segmentation fault.` on this line.